Content.Server/GameObjects/Components/Nutrition/HungerComponent.cs

```diff
@@ -123,7 +129,7 @@ public void HungerThresholdEffect(bool force = false)
                         return;
 
                     case HungerThreshold.Dead:
-                        return;
+                        break;
```

Comment:
What's this change for?

Reply:
You're right. I first wanted to move the StatusEffect set/remove after the switch.